docs/.vuepress/sidebar/en.ts

```diff
@@ -39,14 +39,24 @@ export const enSidebar = sidebar({
       link: "/dev/building-on-zksync",
       children: [
         "/dev/building-on-zksync/hello-world.md",
-        "/dev/building-on-zksync/contracts/contracts.md",
+        "/dev/building-on-zksync/contracts/contract-development.md",
```

Comment:
We should add a redirect to firebase.json due to this change

Reply:
Ok.
Should I cease making such changes?

Reply:
no no... just important to add the redirects indeed :)

Reply:
Should I add these redirects?

Reply:
@kailash-manasarovar I've added redirects. Please review!